### PR TITLE
BOCC keyboard "config.h" had the incorrect amount of LEDs listed. Original code listed 16 rather than 20.

### PR DESCRIPTION
**Pull request by wamsleymj**
## Description

BOCC keyboard "config.h" had the incorrect amount of LEDs listed. Original code listed 16 rather than 20.

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

Evidence from Discord about the issue:
Screen Shot 2021-06-10 at 2 22 08 PM
Screen Shot 2021-06-10 at 2 22 26 PM

Successful `make` output:
[bocc_success_make_output.txt](https://github.com/qmk/qmk_firmware/files/6634624/bocc_success_make_output.txt)

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Comment by wamsleymj** [comment]
@drashna - Thanks for taking a look!

**Comment by wamsleymj** [comment]
So this is approved, but when will it get merged?

**Comment by drashna** [comment]
> So this is approved, but when will it get merged?

Usually, we want 2 approvals before it gets merged.

**Review by fauxpark — APPROVED** [review]

**Pull request merged**
